custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -109,11 +108,9 @@ def get_person_case_structure(self):
                     'dob_known': 'no',
                     'first_name': self.patient_detail.first_name,
                     'last_name': self.patient_detail.last_name,
-                    'middle_name': self.patient_detail.middle_name,
                     'name': self.patient_detail.pname,
                     'nikshay_id': self.nikshay_id,
-                    'permanent_address_district_choice': self.patient_detail.Dtocode,
-                    'permanent_address_state_choice': self.patient_detail.scode,
+                    'person_id': 'FROM_NIKSHAY_' + self.nikshay_id,
```

Comment:
yeah sorry i saw that thread. not sure kriti counts and this seems a little ridiculous to show to users